mmv1/products/networkmanagement/ConnectivityTest.yaml

```diff
@@ -53,6 +53,37 @@ examples:
       network: 'connectivity-vpc'
       source_cloudrun: 'src-cloudrun-v2'
       dest_cloudrun: 'dest-cloudrun-v2'
+  - !ruby/object:Provider::Terraform::Examples
```

Comment:
looks like it's having trouble building the downstream provider. Could you address the issues that prevent the build from happening?
logs:
```make
bundler: failed to load command: compiler.rb (compiler.rb)
Step #11 - "tpgb-head": /mm-terraform-beta-head/mmv1/provider/terraform/examples.rb:214:in `block in config_test_body': undefined method `include?' for true:TrueClass (NoMethodError)
Step #11 - "tpgb-head": 	from /mm-terraform-beta-head/mmv1/provider/terraform/examples.rb:212:in `each'
Step #11 - "tpgb-head": 	from /mm-terraform-beta-head/mmv1/provider/terraform/examples.rb:212:in `map'
Step #11 - "tpgb-head": 	from /mm-terraform-beta-head/mmv1/provider/terraform/examples.rb:212:in `config_test_body'
Step #11 - "tpgb-head": 	from /mm-terraform-beta-head/mmv1/provider/terraform/examples.rb:194:in `config_test'
Step #11 - "tpgb-head": 	from (erb):114:in `block in provider_binding'
Step #11 - "tpgb-head": 	from (erb):40:in `each'
Step #11 - "tpgb-head": 	from (erb):40:in `provider_binding'
Step #11 - "tpgb-head": 	from /usr/local/lib/ruby/3.1.0/erb.rb:905:in `eval'
Step #11 - "tpgb-head": 	from /usr/local/lib/ruby/3.1.0/erb.rb:905:in `result'
Step #11 - "tpgb-head": 	from /mm-terraform-beta-head/mmv1/compile/core.rb:199:in `compile_string'
Step #11 - "tpgb-head": 	from /mm-terraform-beta-head/mmv1/compile/core.rb:141:in `compile_file'
Step #11 - "tpgb-head": 	from /mm-terraform-beta-head/mmv1/provider/file_template.rb:66:in `block in generate'
Step #11 - "tpgb-head": 	from /mm-terraform-beta-head/mmv1/provider/file_template.rb:66:in `open'
Step #11 - "tpgb-head": 	from /mm-terraform-beta-head/mmv1/provider/file_template.rb:66:in `generate'
Step #11 - "tpgb-head": 	from /mm-terraform-beta-head/mmv1/provider/terraform.rb:784:in `generate_resource_tests'
Step #11 - "tpgb-head": 	from /mm-terraform-beta-head/mmv1/provider/terraform.rb:397:in `generate_object'
Step #11 - "tpgb-head": 	from /mm-terraform-beta-head/mmv1/provider/terraform.rb:381:in `block in generate_objects'
Step #11 - "tpgb-head": 	from /mm-terraform-beta-head/mmv1/provider/terraform.rb:364:in `each'
Step #11 - "tpgb-head": 	from /mm-terraform-beta-head/mmv1/provider/terraform.rb:364:in `generate_objects'
Step #11 - "tpgb-head": 	from /mm-terraform-beta-head/mmv1/provider/terraform.rb:94:in `generate'
Step #11 - "tpgb-head": 	from compiler.rb:279:in `block in <top (required)>'
Step #11 - "tpgb-head": 	from /usr/local/bundle/gems/parallel-1.22.1/lib/parallel.rb:587:in `call_with_index'
Step #11 - "tpgb-head": 	from /usr/local/bundle/gems/parallel-1.22.1/lib/parallel.rb:557:in `process_incoming_jobs'
Step #11 - "tpgb-head": 	from /usr/local/bundle/gems/parallel-1.22.1/lib/parallel.rb:537:in `block in worker'
Step #11 - "tpgb-head": 	from /usr/local/bundle/gems/parallel-1.22.1/lib/parallel.rb:528:in `fork'
Step #11 - "tpgb-head": 	from /usr/local/bundle/gems/parallel-1.22.1/lib/parallel.rb:528:in `worker'
Step #11 - "tpgb-head": 	from /usr/local/bundle/gems/parallel-1.22.1/lib/parallel.rb:519:in `block in create_workers'
Step #11 - "tpgb-head": 	from /usr/local/bundle/gems/parallel-1.22.1/lib/parallel.rb:518:in `each'
Step #11 - "tpgb-head": 	from /usr/local/bundle/gems/parallel-1.22.1/lib/parallel.rb:518:in `each_with_index'
Step #11 - "tpgb-head": 	from /usr/local/bundle/gems/parallel-1.22.1/lib/parallel.rb:518:in `create_workers'
Step #11 - "tpgb-head": 	from /usr/local/bundle/gems/parallel-1.22.1/lib/parallel.rb:457:in `work_in_processes'
Step #11 - "tpgb-head": 	from /usr/local/bundle/gems/parallel-1.22.1/lib/parallel.rb:294:in `map'
Step #11 - "tpgb-head": 	from compiler.rb:154:in `<top (required)>'
Step #11 - "tpgb-head": 	from /usr/local/bundle/gems/bundler-2.4.6/lib/bundler/cli/exec.rb:58:in `load'
Step #11 - "tpgb-head": 	from /usr/local/bundle/gems/bundler-2.4.6/lib/bundler/cli/exec.rb:58:in `kernel_load'
Step #11 - "tpgb-head": 	from /usr/local/bundle/gems/bundler-2.4.6/lib/bundler/cli/exec.rb:23:in `run'
Step #11 - "tpgb-head": 	from /usr/local/bundle/gems/bundler-2.4.6/lib/bundler/cli.rb:491:in `exec'
Step #11 - "tpgb-head": 	from /usr/local/bundle/gems/bundler-2.4.6/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
Step #11 - "tpgb-head": 	from /usr/local/bundle/gems/bundler-2.4.6/lib/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
Step #11 - "tpgb-head": 	from /usr/local/bundle/gems/bundler-2.4.6/lib/bundler/vendor/thor/lib/thor.rb:392:in `dispatch'
Step #11 - "tpgb-head": 	from /usr/local/bundle/gems/bundler-2.4.6/lib/bundler/cli.rb:34:in `dispatch'
Step #11 - "tpgb-head": 	from /usr/local/bundle/gems/bundler-2.4.6/lib/bundler/vendor/thor/lib/thor/base.rb:485:in `start'
Step #11 - "tpgb-head": 	from /usr/local/bundle/gems/bundler-2.4.6/lib/bundler/cli.rb:28:in `start'
Step #11 - "tpgb-head": 	from /usr/local/bundle/gems/bundler-2.4.6/exe/bundle:45:in `block in <top (required)>'
Step #11 - "tpgb-head": 	from /usr/local/bundle/gems/bundler-2.4.6/lib/bundler/friendly_errors.rb:117:in `with_friendly_errors'
Step #11 - "tpgb-head": 	from /usr/local/bundle/gems/bundler-2.4.6/exe/bundle:33:in `<top (required)>'
Step #11 - "tpgb-head": 	from /usr/local/bundle/bin/bundle:25:in `load'
Step #11 - "tpgb-head": 	from /usr/local/bundle/bin/bundle:25:in `<main>'
Step #11 - "tpgb-head": make[1]: *** [GNUmakefile:64: mmv1] Error 1
Step #11 - "tpgb-head": make: *** [GNUmakefile:60: provider] Error 2
```
```

Reply:
my bad @BBBmau , I accidentally set `deletion_protection` in the spec to a bool whereas a string is required. I pushed a fix